js/capture.js

```diff
@@ -10,10 +10,22 @@ const capture = {
   addListeners() {
     // listen for each HTTP response
     browser.webRequest.onResponseStarted.addListener(
-      this.sendThirdParty,
+      this.sendThirdParty.bind(this),
```

Comment:
I like the arrow functions better too. Lexical scope makes sense, and you can see the arguments passed in too. Changed!